app/actions/remote/user.ts

```diff
@@ -287,6 +289,41 @@ export const fetchStatusInBatch = (serverUrl: string, id: string) => {
     return debouncedFetchStatusesByIds.apply(null, [serverUrl]);
 };
 
+const mentionNames = new Set<string>();
+export const fetchUserOrGroupsByMentionsInBatch = (serverUrl: string, mentionName: string) => {
+    mentionNames.add(mentionName);
+    return debouncedFetchUserOrGroupsByMentionNames.apply(null, [serverUrl]);
+};
+const debouncedFetchUserOrGroupsByMentionNames = debounce((serverUrl: string) => {
+    fetchUserOrGroupsByMentionNames(serverUrl, Array.from(mentionNames));
+}, 200, false, () => {
+    mentionNames.clear();
+});
```

Comment:
```suggestion
const debouncedFetchUserOrGroupsByMentionNames = debounce(
    (serverUrl: string) => {
        fetchUserOrGroupsByMentionNames(serverUrl, Array.from(mentionNames));
    },
    200,
    false,
    () => {
        mentionNames.clear();
    },
);
```
0/5, just feels more readable.